.evg.yml

```diff
@@ -124,13 +124,21 @@ tasks:
             set -e
             echo "running client tests"
             source ./creds
+
+            mkdir .npm-global
+            GLOBALPREFIX=`pwd`/.npm-global
+            npm config set prefix "$GLOBALPREFIX"
+            export PATH="$GLOBALPREFIX/bin:$PATH"
+            echo $PATH
+
             chown -R $USER stitch-js-sdk
             cd stitch-js-sdk
             curl -OL https://yarnpkg.com/latest.tar.gz
             tar zvxf latest.tar.gz
             mv yarn-* yarn
             export PATH="`pwd`/yarn/bin:$PATH"
-            yarn install
+
+            npm install
```

Comment:
I think you could leave this as `yarn install` and just remove the `package-lock.json` and replace it with the resulting `yarn-lock` from a `yarn install`